src/styles/main.scss

```diff
@@ -1,3 +1,24 @@
-body {
+body,
+h2,
+p,
+div,
+a,
+header,
+nav,
+img,
+ul,
+li {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
```

Comment:
Tutaj chcemy zrobić coś podobnego do grida. Czyli, jeśli mamy 7 kartek albo mniej, to piąta kartka musi być na tym samym poziomie. U ciebie jest inaczej :)
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/32794096/222656319-8b60e214-ec0e-4533-8b0a-e9a8b4ac9ed8.png">